.azure-pipelines/packages.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       solution: Datadog.Trace.proj
       platform: $(buildPlatform)
       configuration: $(buildConfiguration)
-      msbuildArguments: /t:msi /p:InstallerVersion=%GitVersion_MajorMinorPatch%;OutputName=datadog-dotnet-apm-%GitVersion_MajorMinorPatch%-$(buildPlatform);RunWixToolsOutOfProc=true
+      msbuildArguments: /t:msi /p:RunWixToolsOutOfProc=true
       maximumCpuCount: true
 
   - task: PublishPipelineArtifact@0
```

Comment:
The `targetPath` below will need updating. Can you update with the relative directory you saw in local testing?